src/constants/api-params.ts

```diff
@@ -1,3 +1,3 @@
 export const API_URL = "https://api.joinsafedeal.com";
-export const LISTS_URL = "https://joinsafedeal.com/lists";
+export const LISTS_URL = "https://www.joinsafedeal.com/lists";
```

Comment:
This feels like the wrong fix, the fix should be to make the domain work without www as well, www subdomain should only be an alias, it should work without it.

Thoughts, @obender ?

Reply:
True. We need to add an A record for www pointing towards @ in DNS settings. Then it should be able to handle request without www. Right now it permanently redirects all the requests without www to homepage. I had to do this to make lists work for now.